src/content/libraries/config.ts

```diff
@@ -22,6 +22,27 @@ const categories = [
   "utils",
 ] as const;
 
+export const categoryNames: { [key in (typeof categories)[number]]: string } = {
```

Comment:
Is there a reason why these can't be used in their directly renderable form in the frontmatter? I ask because this adds another place for translations when the translations could happen in the MDX if these were stored directly. see #230 for a related question about tutorials

Reply:
I think the main thing we get by using slugs in each entry and then having translations of those is that we get easier type checking, since the options are the same for every locale folder, and then there's one less field that actually needs to be translated when new entries are made (the categories would just need to be translated once upfront.) So I've been thinking of them so far as something closer to UI translations than content translations.

That said, you're right that it's a new thing that needs to be translated. We can definitely still make it work with this being written ready to render in each file. We might need to do something sorta custom to make sure there aren't typos or discrepancies in the categories via Zod though -- is there a good way to do that without having a list of all the translations also stored somewhere so we can enforce those as options? I guess having that list centrally defeats the purpose of translating it in each file?

Reply:
If you think the UI content approach makes sense, I can transition this + the tutorial category names into that file

Reply:
That makes sense to me. I agree that it would be ideal to keep these as enum-like slugs. My concern was primarily about getting too spread out with translations. I think that moving them to a central location should suffice to make it clear that these utilities also need translations. Perhaps we can even do something similar for #184

Reply:
That works for a way forward for #184! I just added some commits moving the library and tutorial category display names to the UI file.